src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml

```diff
@@ -73,44 +73,43 @@
       VulnerabilitiesCheckChanged="CheckboxVulnerabilties_CheckChanged"
       SourceRepoListSelectionChanged="SourceRepoList_SelectionChanged" />
 
-    <Border
+            <Border
       DockPanel.Dock="Top"
       BorderThickness="0,1,0,0"
       BorderBrush="{DynamicResource {x:Static SystemColors.ActiveBorderBrushKey}}"
       VerticalAlignment="Stretch">
-        <Grid
+                <Grid
           Width="{Binding ActualWidth, ElementName=_root}">
-          <Grid.ColumnDefinitions>
-            <ColumnDefinition
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition
               x:Name="_leftSideGridColumn"
-              Width="65*"
-              MinWidth="300">
-              <ColumnDefinition.MaxWidth>
-                <MultiBinding Converter="{StaticResource SubstractionConverter}">
-                  <Binding Path="ActualWidth" ElementName="_root" />
-                  <Binding Path="Width" ElementName="_gridSplitter" />
-                  <Binding Path="MinWidth" ElementName="_rightSideGridColumn" />
-                </MultiBinding>
-              </ColumnDefinition.MaxWidth>
-            </ColumnDefinition>
-            <ColumnDefinition Width="auto" />
-            <ColumnDefinition
+              Width="67*">
```

Comment:
I'm only seeing a couple minor changes to values here, but this diff is large since there's so many whitespace changes. Please revert all the whitespace changes in this file.